### PR TITLE
doc: nrf5340audio: edit two gateway lines

### PR DESCRIPTION
**Pull request by greg-fer**
Edited lines about two gateways.
These were originally introduced in PR#9731.
OCT-2393.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

**Comment by alexsven** [comment]
Will conflict with: https://github.com/nrfconnect/sdk-nrf/pull/9829 but I can rebase after this is merged

File: applications/nrf5340_audio/README.rst [hunk 1]
```diff
@@ -780,12 +780,17 @@ Selecting the BIS mode
...
-   The gateway firmware must be built twice to give two separate gateway devices two separate names (``CONFIG_BT_DEVICE_NAME`` and ``CONFIG_BT_DEVICE_NAME_ALT``).
-   Program the two firmware versions to two separate gateway devices.
+Enabling the BIS mode with two gateways
+---------------------------------------
+
+In addition to the standard BIS mode with one gateway, you can also add a second gateway device that the BIS headsets can receive audio stream from.
+To configure the second gateway, add both the :kconfig:option:`CONFIG_TRANSPORT_BIS` and the :kconfig:option:`CONFIG_BT_USE_DEVICE_NAME_ALT` Kconfig option set to ``y`` to the :file:`prj.conf` file for the debug version and to the :file:`prj_release.conf` file for the release version.
+You can provide an alternative name to the second gateway using the :kconfig:option:`CONFIG_BT_DEVICE_NAME_ALT` or use the default alternative name.
+
+You build the BIS gateways using the normal procedures from :ref:`nrf53_audio_app_building`, but the gateway firmware must be built twice to give two separate gateway devices two separate names (``CONFIG_BT_DEVICE_NAME`` and ``CONFIG_BT_DEVICE_NAME_ALT``).
```
Comment (alexsven) [inline applications/nrf5340_audio/README.rst:792]: This is a confusing sentence. To be clear: If you want an additional gateway that you can sync to, build with CONFIG_BT_USE_DEVICE_NAME_ALT=y and flash to a second gateway. 

Now it seems that you can just build the same configuration twice and it's magically two different gateways

**Review by alexsven — CHANGES_REQUESTED** [review]

File: applications/nrf5340_audio/README.rst (hunk 1 above)
Comment (erikrobstad) [reply applications/nrf5340_audio/README.rst:792]: Hmmm the next sentence is "Remember to program the two firmware versions to two separate gateway devices.", which to me makes it pretty clear that two different gateway devices are needed

Comment (greg-fer) [reply applications/nrf5340_audio/README.rst:792]: Checking with @erikrobstad

File: applications/nrf5340_audio/README.rst [hunk 2]
```diff
@@ -1288,8 +1293,7 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 * Pressing the **PLAY/PAUSE** button on the gateway will respectively start or stop the stream for all headsets listening in.
 * Pressing the **BTN 4** button on a headset will change the active audio stream.
   The default configuration of the BIS mode supports two audio streams (left and right).
-* Pressing the **BTN 5** button on a headset will change the gateway source for the audio stream.
-  The default configuration of the BIS mode supports two gateways (``CONFIG_BT_DEVICE_NAME`` and ``CONFIG_BT_DEVICE_NAME_ALT``).
+* Pressing the **BTN 5** button on a headset will change the gateway source for the audio stream (after `Enabling the BIS mode with two gateways`_).
```
Comment (erikrobstad) [inline applications/nrf5340_audio/README.rst:1296]:
```suggestion
* Pressing the **BTN 5** button on a headset will change the gateway source for the audio stream (after `Enabling the BIS mode with two gateways`_). If a second gateway is not present the headset will not play audio.
```

When BTN 5 is pressed the headset will try to scan for the second gateway even if it is not there (even if we have not done the steps in `Enabling the BIS mode with two gateways`).

File: applications/nrf5340_audio/README.rst (hunk 1 above)
Comment (alexsven) [reply applications/nrf5340_audio/README.rst]: Latest changes look better, resolved

Comment (erikrobstad) [reply applications/nrf5340_audio/README.rst]: Ah, you probably meant that it is not mentioned in this sentence that CONFIG_BT_USE_DEVICE_NAME_ALT=y has to be added for the second gateway

**Review by erikrobstad — APPROVED** [review]

**Comment by greg-fer** [comment]
@alexsven , can you check if the edits are ok for you?

**Review by alexsven — APPROVED** [review]
Sorry, only resolved the conversation, approved!

**Review by rick1082 — APPROVED** [review]

**Pull request merged**